lib/pagseguro/authorization/response.rb

```diff
@@ -14,6 +14,14 @@ def serialize
         end
       end
 
+      def serialize_collection
```

Comment:
@vitorkusiaki You can use the "new way" to set errors, like this merged PR: https://github.com/pagseguro/ruby/pull/111/files#diff-8f8da9b5a8496e78f4e6f822d5f87f6fR9 :)